packages/devextreme-react-generator/src/generator.ts

```diff
@@ -273,6 +326,28 @@ export function mapWidget(
   const propTypings = extractPropTypings(raw.options, customTypeHash)
     .filter((propType) => propType !== null) as IPropTyping[];
 
+  const customTypeImports: Record<string, Array<string>> = {};
+  const defaultTypeImports: Record<string, string> = {};
+  const wildcardTypeImports: Record<string, string> = {};
+
+  widgetCustomTypes.forEach((t) => {
+    if (typeGenerationOptions?.importOverridesMetadata?.defaultImports?.[t]) {
```

Comment:
**Minor** If you extract a variable, you can reuse down the code
```suggestion
     var importOverrides = (typeGenerationOptions?.importOverridesMetadata;
```

Reply:
fixed